client/src/App.tsx

```diff
@@ -19,11 +18,10 @@ export const store = createStore(
 const App: React.FC = (): ReactElement => {
     return (
         <Provider store={store}>
-                <AlertContainer/>
-                {routes}
-                <Footer/>
+        <AlertContainer/>
```

Comment:
prev is better. They are children. They should be nested

Reply:
fixed